mysql/datadog_checks/mysql/mysql.py

```diff
@@ -93,6 +93,7 @@ def __init__(self, name, init_config, instances):
         self._agent_hostname = None
         self._is_aurora = None
         self._config = MySQLConfig(self.instance)
+        self._only_custom_queries = is_affirmative(self.instance.get('only_custom_queries', False))
```

Comment:
> self._only_custom_queries = is_affirmative(self.instance.get('only_custom_queries', False))

nit; I think this should be in `MySQLConfig`?

So it would be under `self._config.only_custom_queries`

Reply:
Updated!